libraries/Microsoft.Bot.Builder/Skills/SkillHandlerImpl.cs

```diff
@@ -0,0 +1,186 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Security.Claims;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Bot.Schema;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+
+namespace Microsoft.Bot.Builder.Skills
+{
+    internal class SkillHandlerImpl
```

Comment:
Add a brief summary describing what the class do

Reply:
Added summary.